sci-libs/hdf5/metadata.xml

```diff
@@ -6,13 +6,14 @@
     <name>Gentoo Science Project</name>
   </maintainer>
   <longdescription>
-  HDF5 is a file format and library for storing scientific data. HDF5
-  was designed and implemented to address the deficiencies of HDF4.x. It
-  has a more powerful and flexible data model, supports files larger
-  than 2 GB, and supports parallel I/O.
-</longdescription>
+    HDF5 is a file format and library for storing scientific data. HDF5
+    was designed and implemented to address the deficiencies of HDF4.x. It
+    has a more powerful and flexible data model, supports files larger
+    than 2 GB, and supports parallel I/O.
+  </longdescription>
   <use>
     <flag name="hl">Enable high level API (https://support.hdfgroup.org/HDF5/doc/HL/index.html)</flag>
+    <flag name="ros3">Enable support for read-only S3 access (https://portal.hdfgroup.org/display/HDF5/Configuration+and+Setup+for+HDF5+Read+Only+S3+VFD)</flag>
```

Comment:
I can't access this link?  
https://portal.hdfgroup.org/display/HDF5/Configuration+and+Setup+for+HDF5+Read+Only+S3+VFD

Reply:
@jsmolic I tried again, and the link seems to be working...

Reply:
Yep, works for me now as well